interactors/keyboard.js

```diff
@@ -9,6 +9,38 @@ export default createInteractor('keyboard (requires window to have focus)')({
     arrowDown: press('ArrowDown'),
     arrowLeft: press('ArrowLeft'),
     arrowRight: press('ArrowRight'),
+    // using an underscore to signify that this is a temporary workaround
+    // we want to use the default browser implementation, but it doesn't
+    // trigger the built-in browser functions so we will need a more
+    // fleshed out approach to this
+    _nextOption: perform(async (el) => {
```

Comment:
If we ask folks to use it, then it's a part of the public API and so we should just go ahead and own it, and then deprecate it when we have a better solution.

Reply:
That is fair. Do you then think I should split it into a `<select>` and a `<input type="number">` version instead of the overloaded version I have here?